.github/workflows/release.yaml

```diff
@@ -81,3 +88,32 @@ jobs:
               env:
                 GITHUB_TOKEN: ${{ secrets.CODEFLARE_MACHINE_ACCOUNT_TOKEN }}
               shell: bash
+
+            - name: Append tag to Snyk monitoring list
+              run: |
+                sed -i 's/list_of_released_tags=(/list_of_released_tags=("v${{ github.event.inputs.release-version }}", /' .github/workflows/snyk-security.yaml
+
+            - name: Commit and push changes
+              run: |
+                git config --global user.email "138894154+codeflare-machine-account@users.noreply.github.com"
+                git config --global user.name "codeflare-machine-account"
```

Comment:
Can we make these variables at the repo level and pass by reference here instead that way if they ever change we only have to update GH config?